index.md

```diff
@@ -1,12 +1,45 @@
-# Welcome!
+# Welcome & Congratulations!
+### You just took your first step towards becoming a Virtual Intern!
 
-This MDwiki serves as an information repository for treehouses, a project of the Open Learning Exchange.  Please feel free to join us on our [Gitter chat](https://gitter.im/treehouses/Lobby).
+<br />
 
-The treehouses project is divided into several parts to meet the needs of different users, including those participating in our Virtual Intern program.
+![OLE International Team](./images/OLE_International_Team.jpg)
+###### The OLE International Team. [Left to Right] Rabi, April Stone, Stefan Unterhauser, Rory Phimister, Denisia Mangue, Richard R. Rowe, Boyah J. Farah and Leonard Maximus Ireneous Mensah not pictured.
 
-If you are a new Virtual Intern, please go to the "Virtual Interns" tab and go to "Introduction / First Steps", or [click here](pages/vi/firststeps.md) to get started!  Please remember that many team members are available to help you on Gitter chat.
+<br />
```

Comment:
this is not up to date ... lets just remove it ;)

Reply:
What do you mean? I believe I changed that portion also.